tests/integration/helpers/cluster.py

```diff
@@ -1521,7 +1528,7 @@ def start(self, destroy_dirs=True):
 
             if self.with_mysql_client and self.base_mysql_client_cmd:
                 logging.debug('Setup MySQL Client')
-                subprocess_check_call(self.base_mysql_client_cmd + common_opts)
+                subprocess_check_call(self.base_mysql_client_cmd + common_opts, num_tries=DOCKER_COMPOSE_CMD_NUM_TRIES)
```

Comment:
Interesting, there was already pull with retries, but this had been removed in 10b7037d9c7, in the merge commit, maybe @qoega can comment on this?

But AFACIS it will not pull images with separate `docker-compose.yml` files, i.e. mysql and similar.